src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json

```diff
@@ -78,6 +78,7 @@
 					"mediaElement": "false",
 					"navigation": "blank",
 					"platforms": "android|ios|maccatalyst|wasm|windows|desktop",
+          "rider": "true",
```

Comment:
What does this parameter do? Similar to IsVsix, if this is specific to Rider, it shoudln't be in here

Reply:
So this actually defaults to true on the CLI. It adds a `.rider` folder with some configurations that allow you to run the project in Rider... similar to the VS Code stuff we have.

Reply:
Agreed. VSIX needs to set isVsix=true and rider=false as overrides to this file